docs/.vuepress/theme/NotFound.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div id="wrapper" :class="pageClasses">
-    <Header @toggle-mode="toggleMode" />
+    <Navigation @toggle-mode="toggleMode" />
```

Comment:
Let's change this to `dark-mode-toggle` to match your changes to Layout.vue - I think this is broken right now.